B827EBFFFE85273B.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE85273B",
+    "servers": [
+      {
+        "server_address": "ttn-router-us-west",
```

Comment:
Not a valid router address. please check the list here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses

Reply:
Corrected and pulled again